index.rst

```diff
@@ -21,3 +20,16 @@ This tutorial was last given at SciPy 2020 in Austin Texas.
    03_dask.delayed
    04_distributed
    05_futures
+
+
+More tutorials from our community
```

Comment:
A section like this could also make sense in https://docs.dask.org/en/latest/presentations.html

Reply:
Yeah totally agree. I think it would be good to differentiate between folks "talking about Dask wearing a Dask hat" and "talking about Dask in another context" in that section.

For example a maintainer talking about maintenance should be clearly Dask content. A project like Xarray talking about how they use Dask should clearly be external content. Coiled running Dask tutorials should also clearly be external content.